realm/src/main/java/io/realm/RealmQuery.java

```diff
@@ -73,6 +73,7 @@ public RealmQuery(Realm realm, Class<E> clazz) {
         this.realm = realm;
         this.clazz = clazz;
         this.table = realm.getTable(clazz);
+        view = null;
```

Comment:
No need since it's the default value for fields extending `Object`. Being explicit about it might be a good idea and will probably be optimized away from the compiler, but still... I don't think this should be here. What does the rest of @realm/java think?